packages/block-editor/src/default-editor-styles.scss

```diff
@@ -0,0 +1,9 @@
+body {
+	font-family: $default-font;
+	line-height: 1.5;
+	--wp--style--block-gap: 1.5em;
+}
+
+p {
+	line-height: 2;
+}
```

Comment:
@jasmussen this is the styles file if you want to tweak it.

Reply:
I've got a related PR at https://github.com/WordPress/gutenberg/pull/34334 that enqueues the global styles for all themes. After it lands, shouldn't these styles live at `lib/theme.json`?

Reply:
I don't know, what I know is that they need to be in a dedicated setting in order for this logic to apply https://github.com/WordPress/gutenberg/pull/34439/files#diff-d00cfe048041886ee9b16f6f42422317b6e770cbea20cbe0e4d0ca7c8e0ec0e1R159-R161